kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -90,18 +80,25 @@
     private DeactivatedBeneficiaryService deactivatedBeneficiaryService;
 
     private static final Integer REJECTION_PART_SIZE = 5000;
```

Comment:
Is there some property/constants file -- to put rejection_part_size and add comment on what it is